src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/custom.py

```diff
@@ -1424,9 +1424,9 @@ def create_vm(vm_name, resource_group_name, image=None,
               public_ip_address=None, public_ip_address_allocation='dynamic',
               public_ip_address_dns_name=None,
               os_disk_name=None, os_type=None, storage_account=None,
-              storage_caching='ReadWrite', storage_container_name='vhds',
+              storage_caching=None, storage_container_name=None,
```

Comment:
Yes, it is. The reason is because since these are in the list of "forbidden" parameters for certain profiles, then having a default means that these always throw.  I toyed with using the "MarkSpecifiedAction" on these, but that always seemed like a hacky workaround (and it has demonstrably introduced bugs) and this was a simpler solution.  Previously, we silently ignored these parameters when they were defaulted or specified, but we have said that we do not want to do that.